src/Stripe.Tests/transfers/when_creating_a_transfer_with_application_fee.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ***** This test needs a bank account setup in order to run correctly, and the ignore attribute wasn't working. Good times.
+
+//using Machine.Specifications;
+
+//namespace Stripe.Tests
+//{
+//    public class when_creating_a_transfer_with_application_fee
+//    {
+//        protected static StripeTransferCreateOptions StripeTransferCreateOptions;
+//        protected static StripeTransfer StripeTransfer;
+
+//        private static StripeTransferService _stripeTransferService;
+
+//        Establish context = () =>
+//        {
+//            _stripeTransferService = new StripeTransferService();
+//            StripeTransferCreateOptions = test_data.stripe_transfer_create_options.Valid();
```

Comment:
This can just be declared inline....  var createOptions = ....  Since it's not used anywhere else. The reason for having them protected at the top is so the behavior can access them for assertions. Make sense?